aws/README.md

```diff
@@ -161,10 +161,99 @@ To push a new release of the data service, follow the same procedure but change
 
 You can list the existing tags/versions with `git tag` or on the [github repo](https://github.com/open-covid-data/healthmap-gdo-temp/releases).
 
+### `Latest` image tag for dev
+
+Dev instances of curator and data services are using the `latest` image tag, that's not best practice but is okay while we work on other features. The latest image is fetched when a deployment is updated, to update dev to the `latest` image built by docker-hub, do:
+
+```shell
+kubectl rollout restart deployment/curator-dev
+kubectl rollout restart deployment/curator-prod
+```
+
 ### Rollback
 
 Just change the image tag referenced in the deployment file to an earlier version and apply the change.
 
 ### Deleting a release
 
-If for some reason you need to delete a tag, you can do it with `git tag -d curator-1.2.3` then `git push origin :refs/tags/curator-0.1.2` to delete it remotely.
\ No newline at end of file
+If for some reason you need to delete a tag, you can do it with `git tag -d curator-1.2.3` then `git push origin :refs/tags/curator-0.1.2` to delete it remotely.
+
+## Ingress / Application load balancer
+
+Curator services are exposed here:
+
+- [dev](http://51cfaa79-default-curatorde-771d-764651811.us-east-1.elb.amazonaws.com/)
+- [prod](http://51cfaa79-default-curatorpr-5c7d-1665373250.us-east-1.elb.amazonaws.com)
+
+TODO: Apply `external-dns` to configs once we know where we'll run those.
+
+TODO: Map port 443 in ingress configs for HTTPS support.
+
+This section unfortunately involves quite a few sequential command-line executions for properly setting-up IAM.
+
+The setup was done following a mix of articles:
+
+- https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html
+- https://aws.amazon.com/premiumsupport/knowledge-center/eks-alb-ingress-controller-fargate/
+- https://kubernetes-sigs.github.io/aws-alb-ingress-controller/guide/walkthrough/echoserver/
+- https://aws.amazon.com/blogs/containers/using-alb-ingress-controller-with-amazon-eks-on-fargate/
+
+
+```shell
```

Comment:
Do these commands need to be run again in the future? If so, could you describe why/when they should be run? If not, maybe we don't need to keep them in the README - up to you.

Reply:
They would need to be re-run if we re-create the cluster (which I don't expect to happen but it could) which is why I've documented them here.
I've added a bit of documentation.